code/modules/awaymissions/mission_code/ghost_role_spawners/golems.dm

```diff
@@ -134,6 +134,27 @@
 		user.death()
 		return
 
+/obj/effect/mob_spawn/human/golem/attackby(obj/item/slimepotion/transference/I, mob/living/carbon/user, params)
```

Comment:
Not 100% a fan of checking for the item's type this way, because if someone wanted to add another attack behaviour for another item, they wouldn't be able to. Better to leave the argument as `obj/item/I` and pass the desired type in `istype`'s second argument in a `if`/`else if`